### PR TITLE
Extract new strings (2024-08-29)

### PR DESCRIPTION
**Pull request by mozilla-pontoon**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

File: templates/firefox-ios.xliff [hunk 1]
```diff
@@ -3250,6 +3250,49 @@ Private Tab</source>
...
+      <trans-unit id="Bookmarks.Menu.BookmarksTopLevelTitle.v131" xml:space="preserve">
+        <source>Bookmarks</source>
+        <note>Name of the top level bookmarks folder present in the folder selection menu of the`Edit Bookmark` screen</note>
+      </trans-unit>
+      <trans-unit id="Bookmarks.Menu.DeleteFolder.v131" xml:space="preserve">
+        <source>Delete Folder</source>
+        <note>When a folder is longpressed in the bookmarks menu, a `Delete Folder` button is present.</note>
+      </trans-unit>
+      <trans-unit id="Bookmarks.Menu.DeletedBookmark.v131" xml:space="preserve">
+        <source>Deleted "%@"</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3271]: Couldn't track down how landed this string for some reason.
@ih-codes Could you help find someone to fix this please?
We would need [typographic](https://acorn.firefox.com/latest/content/punctuation-7J3JVM4o#section-apostrophes-and-quotation-marks-65) quotes in this instance. Thanks!

**Review by Delphine — CHANGES_REQUESTED** [review]

File: templates/firefox-ios.xliff (hunk 1 above)
Comment (flodolo) [reply templates/firefox-ios.xliff:3271]: String are mostly added in one file: https://github.com/mozilla-mobile/firefox-ios/blob/main/firefox-ios/Client/Frontend/Strings.swift

String in question: https://github.com/mozilla-mobile/firefox-ios/blob/main/firefox-ios/Client/Frontend/Strings.swift#L166

PR: github.com/mozilla-mobile/firefox-ios/blob/main/firefox-ios/Client/Frontend/Strings.swift#L166

@DanielDervishi

Comment (ih-codes) [reply templates/firefox-ios.xliff:3271]: Thanks @flodolo, @DanielDervishi just [merged in a fix](https://github.com/mozilla-mobile/firefox-ios/pull/21666) for this.

@Delphine You should be good to pull in the strings again!

File: templates/firefox-ios.xliff [hunk 2]
```diff
@@ -3915,6 +4002,197 @@ Private Tab</source>
...
+      <trans-unit id="MainMenu.TabsSection.NewPrivateTab.Title.v131" xml:space="preserve">
+        <source>New Private Tab</source>
+        <note>On the main menu, the title for the action that will create a new private tab.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.TabsSection.NewTab.Title.v131" xml:space="preserve">
+        <source>New Tab</source>
+        <note>On the main menu, the title for the action that will create a new, non-private, tab.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.ToolsSection.FindInPage.Title.v131" xml:space="preserve">
+        <source>Find in Page...</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:4175]: @adudenamedruby Linter caught this, should be an ellipsis (vs three dots). Thanks!

File: templates/firefox-ios.xliff [hunk 3]
```diff
@@ -3915,6 +4002,197 @@ Private Tab</source>
...
+        <source>Add to Shortcuts</source>
+        <note>On the main menu, in the Save submenu, the title for the menu component that allows a user to add the current website to the shortcuts on the homepage.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Save.BookmarkThisPage.Subtitle.v131" xml:space="preserve">
+        <source>Add Bookmark</source>
+        <note>On the main menu, a string below the Save submenu title, indicating what kind of tools are available in that menu. This string is for the Bookmarks tool.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Save.BookmarkThisPage.Title.v131" xml:space="preserve">
+        <source>Bookmark This Page</source>
+        <note>On the main menu, in the tools submenu, the title for the menu component that indicates the current zoom level. Placeholder is for the current zoom level percentage.</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:4080]: Comment seems off and should be updated.

**Review by Delphine — CHANGES_REQUESTED** [review]
Spotted just a few updates. Thanks!

**Comment by ih-codes** [comment]
> Spotted just a few updates. Thanks!

Thanks @Delphine, @adudenamedruby has merged in his PR for these 2 fixes. 👍

**Review by Delphine — APPROVED** [review]

**Pull request merged**